File: python/trie_hash/Solution1732.py
# 1732. 找到最高海拔
from typing import List


class Solution:
    def largestAltitude(self, gain: List[int]) -> int:
        prefix_sum = [0] * (len(gain) + 1)
        for i in range(0, len(gain)):
            prefix_sum[i + 1] = prefix_sum[i] + gain[i]
        return max(prefix_sum)

File: src/trie_hash/Solution1732.java
// 1732. 找到最高海拔
package trie_hash;

public class Solution1732 {
    public int largestAltitude(int[] gain) {
        int prefixSum = 0;
        int max = 0;
        for (int i = 0; i < gain.length; i++) {
            prefixSum += gain[i];
            max = Math.max(prefixSum, max);
        }

        return max;
    }

}
